src/VAERSFileDownloader.py

```diff
@@ -58,5 +58,5 @@ def downloadVAERSFileAndUnzip(file, workingDirectory):
 
 def updateVAERSFiles(years, workingDirectory):
     for year in years:
-        downloadVAERSFileAndUnzip(f'{year}VAERSData.zip', workingDirectory)
```

Comment:
What is the problem with  f-Strings?

Reply:
Hey just saw your message, the problem was the regular python interpreter doesn't have that, iPython does though